mix.exs

```diff
@@ -56,7 +56,7 @@ defmodule VintageNetWizard.MixProject do
     [
       # Dependencies for all targets
       {:plug_cowboy, "~> 2.0"},
-      {:phoenix_html, "~> 2.13"},
+      {:phoenix_html, "~> 2.14"},
```

Comment:
I'm confused. `~> 2.13` allows you to use version 2.14. You shouldn't have gotten an error. What happened before this change was made?

Reply:
For some reason, mix deps.get complained of the mismatch. Operator error most likely.